CHANGELOG.md

```diff
@@ -4,6 +4,37 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+## 8.12.0
```

Comment:
We should add a contribution note for n4bb12 (https://github.com/getsentry/sentry-javascript/pull/12598)